c1322368.lua

```diff
@@ -41,6 +41,9 @@ function c1322368.spop(e,tp,eg,ep,ev,re,r,rp)
 	local opt=e:GetLabel()
 	if (opt==0 and tc:IsType(TYPE_MONSTER)) or (opt==1 and tc:IsType(TYPE_SPELL)) or (opt==2 and tc:IsType(TYPE_TRAP)) then
 		local zone=e:GetHandler():GetLinkedZone()
```

Comment:
>手札に加えるかこのカードのリンク先となる**自分フィールド**に特殊召喚する。

so it should be `local zone=e:GetHandler():GetLinkedZone(tp)`

Reply:
Updated.